src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/Streaming/DataStreamWriterTests.cs

```diff
@@ -273,6 +282,15 @@ private void WriteCsv(int start, int count, string path)
             }
         }
 
+        private static void FailAfter(TimeSpan timeout, Action func)
```

Comment:
Should this function be private to this class, or treated as a shared utility function?